### PR TITLE
fix checkdoc docstring warnings

### PR DESCRIPTION
**Pull request by progfolio**
Fixes docstring warnings about incorrect quoting, missing parameters, length etc.

### CODE REVIEW

File: org-timeblock.el [hunk 1]
```diff
@@ -683,7 +683,7 @@ commands"
   (quit-window t))
 
 (defun ot--schedule-time (marker eventp)
-  "Change the timestamp of the org entry at MARKER to a specified time(range) interactively.
+  "Interactively change Org entry timestamp at MARKER to a specified time-range.
```
Comment (ichernyshovvv) [inline org-timeblock.el:686]: Please change to "Interactively change time for Org entry timestamp at MARKER."

File: org-timeblock.el [hunk 2]
```diff
@@ -683,7 +683,7 @@ commands"
...
 If EVENTP is non-nil, change timestamp of the event.
 
 Schedule or event date won't be changed.
```
Comment (ichernyshovvv) [inline org-timeblock.el:689]: I think, we need to clarify here that the time might be a timerange which depends on user interactive choice.

File: org-timeblock.el [hunk 3]
```diff
@@ -787,7 +787,8 @@ PROMPT can overwrite the default prompt."
 
 (defun ot-construct-id (&optional marker eventp)
   "Construct identifier for the org entry at MARKER.
-If MARKER is nil, use entry at point."
+If MARKER is nil, use entry at point.
+If EVENTP is non-nil use entry's TIMESTAMP property."
```
Comment (ichernyshovvv) [inline org-timeblock.el:791]: Please add a comma after 'non-nil'

**Review by ichernyshovvv — CHANGES_REQUESTED** [review]

**Comment by progfolio** [comment]
I've addressed most of your comments. I left out a few changes, which I think you'll be better suited to reword as you see fit.

File: org-timeblock.el (hunk 2 above)
Comment (progfolio) [reply org-timeblock.el]: Left for you to reword as you see fit.

**Pull request merged**